sdk/ml/azure-ai-ml/azure/ai/ml/operations/_code_operations.py

```diff
@@ -92,8 +92,13 @@ def create_or_update(self, code: Code) -> Code:
                     registry=self._registry_name,
                     body=get_asset_body_for_registry_storage(self._registry_name, "codes", name, version),
                 )
+            show_progress = kwargs.pop("show_progress", True)
             code, _ = _check_and_upload_path(
-                artifact=code, asset_operations=self, sas_uri=sas_uri, artifact_type=ErrorTarget.CODE
+                artifact=code,
+                asset_operations=self,
+                sas_uri=sas_uri,
+                artifact_type=ErrorTarget.CODE,
+                show_progress=show_progress,
```

Comment:
I think you can do `self._show_progress` here as you get it from `_ScopeDependentOperations` inheritance.